modules/dr-recover-lost-control-plane-hosts.adoc

```diff
@@ -85,7 +85,7 @@ In a terminal that has access to the cluster, run the following command to verif
 ----
 $ oc get nodes -l node-role.kubernetes.io/master
 NAME                                         STATUS   ROLES    AGE   VERSION
-ip-10-0-143-125.us-east-2.compute.internal   Ready    master   46m   v1.13.4+db7b699c3
+ip-10-0-143-125.us-east-2.compute.internal   Ready    master   46m   v1.14.0+f667219f4
```

Comment:
I got,
```
$ oc get nodes 
NAME                                         STATUS   ROLES    AGE     VERSION
ip-10-0-129-61.us-east-2.compute.internal    Ready    worker   176m    v1.14.6+c4799753c
ip-10-0-130-169.us-east-2.compute.internal   Ready    master   4h39m   v1.14.6+c4799753c
ip-10-0-134-195.us-east-2.compute.internal   Ready    worker   4h33m   v1.14.6+c4799753c
ip-10-0-140-215.us-east-2.compute.internal   Ready    worker   176m    v1.14.6+c4799753c
ip-10-0-153-190.us-east-2.compute.internal   Ready    worker   4h33m   v1.14.6+c4799753c
ip-10-0-153-201.us-east-2.compute.internal   Ready    master   4h39m   v1.14.6+c4799753c
ip-10-0-167-172.us-east-2.compute.internal   Ready    master   4h39m   v1.14.6+c4799753c
```

Reply:
Thank you @liangxia! I'll update them all to `v1.14.6+c4799753c` and merge.